modules/concept-docs/pages/analytics-for-sdk-users.adoc

```diff
@@ -23,5 +23,5 @@ For complex and long-running queries, involving large ad hoc join, set, aggregat
 
 == Additional Resources
 
-* Start with our  xref:7.0@server:analytics:primer-beer.adoc[introductory primer].
+* Start with our  xref:7.1@server:analytics:primer-beer.adoc[introductory primer].
```

Comment:
{server-version} in attributes.

Reply:
Sorry, laziness on my part...
(I did these as a find-replace for now, and it's safe to do literal "7.1" whereas the attribute requires you to check that attributes.doc is included)

Reply:
I think we can do the `{version-server}` cleanup thing in a separate PR anyway. 
To avoid adding more scope to this one.